devices/candeo.js

```diff
@@ -0,0 +1,19 @@
+const reporting = require('../lib/reporting');
+const extend = require('../lib/extend');
+
+module.exports = [
+    {
+        zigbeeModel: ['HK-DIM-A'],
+        model: 'HK-DIM-A',
+        vendor: 'Candeo',
+        description: 'ZigBee LED Dimmer Smart Switch',
+        extend: extend.light_onoff_brightness({noConfigure: true, disableEffect: true}),
+        configure: async (device, coordinatorEndpoint, logger) => {
+            await extend.light_onoff_brightness().configure(device, coordinatorEndpoint, logger);
+            const endpoint = device.getEndpoint(1);
+            await reporting.bind(endpoint, coordinatorEndpoint, ['genOnOff', 'genLevelCtrl']);
+            await reporting.onOff(endpoint);
+            await reporting.brightness(endpoint);
```

Comment:
We cannot prevent reporting when controlled through z2m, but we can reduce the interval, change this line to: `await reporting.brightness(endpoint, {min: 3});` (device will report at least with intervals of 3 seconds now)

Reply:
Thanks.

I've tested ```{min: 1}``` and it seems a reasonable compromise when you want to use the reported brightness for automations and don't want a 3 second delay.